webapp/src/main/webapp/templates/freemarker/edit/forms/dateTimeEntryForm.ftl

```diff
@@ -3,13 +3,13 @@
 <fieldset class="dateTime">
   <input type="hidden" id="literal" name="literal" value="${literalValues}" role="input"/>
   <#if datatype?contains("#date") || datatype?contains("Year") >
-    <label for="dateTimeField-year">Year</label>
+    <label for="dateTimeField-year">${i18n()[('label.dateTimeWithPrecision.year_capitalized')]}</label>
     <input class="text-field" name="dateTimeField-year" id="dateTimeField-year" type="text" value="" size="4" maxlength="4" />
   </#if>
   <#if datatype?contains("#date") || datatype?contains("Month") >
-    <label for="dateTimeField-month">Month</label>
+    <label for="dateTimeField-month">${i18n()[('label.dateTimeWithPrecision.month_capitalized')]}</label>
     <select name="dateTimeField-month" id="dateTimeField-month" >
-        <option value="">month</option>
+        <option value="">${i18n()[('label.dateTimeWithPrecision.month_capitalized')]}</option>
```

Comment:
This was lower-cased before. Did you intentionally choose to make it capitalized?
Same is true to others below.

Reply:
Just copying the logic in https://github.com/vivo-project/Vitro/pull/203/files. The strings are already capitalized in the i18n file so not really sure if it's necessary for freemarker to capitalize them again, but it seemed nice to be consistent with the other date time form.